### PR TITLE
Lint E231

### PR DESCRIPTION
**Pull request by goldmermaid**

### CODE REVIEW

File: chapter_computer-vision/anchor.md [hunk 1]
```diff
@@ -184,8 +184,8 @@ Next, we will look at a detailed example. First, construct four anchor boxes. Fo
 anchors = np.array([[0.1, 0.08, 0.52, 0.92], [0.08, 0.2, 0.56, 0.95],
                     [0.15, 0.3, 0.62, 0.91], [0.55, 0.2, 0.9, 0.88]])
 offset_preds = np.array([0] * anchors.size)
-cls_probs = np.array([[0] * 4,  # Predicted probability for background
-                      [0.9, 0.8, 0.7, 0.1],  # Predicted probability for dog
+cls_probs = np.array([[0] * 4, # Predicted probability for background
+                      [0.9, 0.8, 0.7, 0.1], # Predicted probability for dog
```
Comment (astonzhang) [inline chapter_computer-vision/anchor.md]: 2 space before inline comment, not 1 space

File: chapter_computer-vision/fine-tuning.md [hunk 2]
```diff
@@ -38,7 +38,7 @@ npx.set_np()
 
 ### Get the Dataset
 
-The hot dog dataset we use was taken from online images and contains 1,400 positive images containing hot dogs and same number of negative images containing other foods. 1,000 images of various classes are used for training and the rest are used for testing.
+The hot dog dataset we use was taken from online images and contains 1, 400 positive images containing hot dogs and same number of negative images containing other foods. 1,000 images of various classes are used for training and the rest are used for testing.
```
Comment (astonzhang) [inline chapter_computer-vision/fine-tuning.md]: 1,400 is 1400 here, remove space

File: chapter_computer-vision/kaggle-gluon-dog.md [hunk 3]
```diff
@@ -31,7 +31,7 @@ npx.set_np()
 
 ## Obtain and Organize the Dataset
 
-The competition data is divided into a training set and testing set. The training set contains 10,222 images and the testing set contains 10,357 images. The images in both sets are in JPEG format. These images contain three RGB channels (color) and they have different heights and widths. There are 120 breeds of dogs in the training set, including Labradors, Poodles, Dachshunds, Samoyeds, Huskies, Chihuahuas, and Yorkshire Terriers.
+The competition data is divided into a training set and testing set. The training set contains 10, 222 images and the testing set contains 10, 357 images. The images in both sets are in JPEG format. These images contain three RGB channels (color) and they have different heights and widths. There are 120 breeds of dogs in the training set, including Labradors, Poodles, Dachshunds, Samoyeds, Huskies, Chihuahuas, and Yorkshire Terriers.
```
Comment (astonzhang) [inline chapter_computer-vision/kaggle-gluon-dog.md]: same here, can you self-check other similar  issues?

File: chapter_multilayer-perceptrons/kaggle-house-price.md [hunk 4]
```diff
@@ -98,8 +98,8 @@ train_data = pd.read_csv('../data/kaggle_house_pred_train.csv')
 test_data = pd.read_csv('../data/kaggle_house_pred_test.csv')
 ```
 
-The training dataset includes 1,460 examples, 80 features, and 1 label.,
-the test data contains 1,459 examples and 80 features.
+The training dataset includes $1,460$ examples, 80 features, and 1 label.,
```
Comment (astonzhang) [inline chapter_multilayer-perceptrons/kaggle-house-price.md]: add $$

File: chapter_preliminaries/pandas.md [hunk 5]
```diff
@@ -19,11 +19,11 @@ As an example, we begin by creating an artificial dataset that is stored in a cs
...
-    f.write('NumRooms,Alley,Price\n')  # Column names
-    f.write('NA,Pave,127500\n')  # Each row is a data point
-    f.write('2,NA,106000\n')
-    f.write('4,NA,178100\n')
-    f.write('NA,NA,140000\n')
+    f.write('NumRooms, Alley, Price\n')  # Column names
+    f.write('NA, Pave, 127500\n')  # Each row is a data point
+    f.write('2, NA, 106000\n')
+    f.write('4, NA, 178100\n')
+    f.write('NA, NA, 140000\n')
```
Comment (astonzhang) [inline chapter_preliminaries/pandas.md]: revert

**Comment by astonzhang** [comment]
Thank you very much for your continuous improvement to address feedback!
